hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -205,6 +215,18 @@ class HassioAddonInfo extends LitElement {
               `
             : ""}
           <div class="security">
+            <ha-label-badge
```

Comment:
Add a check if `stage` is defined

Reply:
stage is allways defined, it falls back to stable as a default

Reply:
Then your types are wrong